wrapanapi/virtualcenter.py

```diff
@@ -221,7 +221,7 @@ def _get_updated_obj(self, obj):
         filter_ = property_collector.CreateFilter(filter_spec, True)
         update = property_collector.WaitForUpdates(None)
         if not update or not update.filterSet or not update.filterSet[0]:
-            self.logger.warning('No object found when updating %s', str(obj))
+            self.logger.warning('No object found when updating {}'.format(str(obj)))
             return
```

Comment:
I think we want to keep the logger.warning('string string %s', str(obj))  syntax here - @mfalesni comment earlier was for the places where % is being used, instead of passing 2nd argument to the logger method